src/SolutionTemplate/UnoSolutionTemplate.VSIX/UnoPlatformPackage.cs

```diff
@@ -187,8 +189,15 @@ private async Task OnOpenedAsync(CancellationToken ct)
 				{
 					var componentModel = (IComponentModel)await GetServiceAsync(typeof(SComponentModel));
 					IVsPackageInstallerServices installerServices = componentModel.GetService<IVsPackageInstallerServices>();
+
+					// GetInstalledPackages is not async, switch to the background
+					await TaskScheduler.Default;
+
 					var installedPackages = installerServices.GetInstalledPackages();
 
+					// Return to the UI thread for DTE uses
+					await this.JoinableTaskFactory.SwitchToMainThreadAsync(ct);
+
```

Comment:
Why not just `await Task.Run` ?

Reply:
It's a guideline for VS 2022 addins threading.